### ISSUE TITLE
[feature request]Can you make Export API node fully supports Text Tokens?

### ISSUE THREAD

**Issue by MoYiha** [WASasquatch/was-node-suite-comfyui#218]
I have tested with latest version, In Export API node, Text Tokens available in output_path but filename_prefix can not .

Thanks for your work.

**Comment by WASasquatch** [comment]
Are you sure? We are applying token parsing here: https://github.com/WASasquatch/was-node-suite-comfyui/blob/main/WAS_Node_Suite.py#L7010

**Comment by MoYiha** [comment]
Yes, I'm sure. Text Tokens with Save Text File and Save Image nodes are works fine, but Export API node is not.
Is https://github.com/WASasquatch/was-node-suite-comfyui/blob/main/WAS_Node_Suite.py#L7010 in was_save_images function right?

**Comment by WASasquatch** [comment]
Ohh my apologies, yeah that's possible. Probably should be a toggle though.
